#163388128 Setup and deploy base app to heroku and upgrade Django

PaulinhoNdegwa opened the pull request:
**What does this PR do?**
- Upgrade Django from version 1.0 to 2.1
- Sets up and deploys the base app to Heroku

**Description of Task to be completed?**
- Upgrade Django from version 1.0 to 2.1
- Created a pipeline on Heroku with review apps, staging, and production
Configured app.json file

**How should this be manually tested?**
Here are the app URLs:

Staging-https://ah-the-phoenix-staging.herokuapp.com/
Production-https://ah-the-phoenix.herokuapp.com/

**Related PT story**
#163388128

Commit:
chore(postgresql):Integrate posgressl Database engine
- configure postgres database engine
- create travis.yml
[starts #163388127]

dev-jey commented:
Hi Paul, great work. You may change the formatting of the PR though. For instance, the titles should not be in italics.

PaulinhoNdegwa commented:
> Hi Paul, great work. You may change the formatting of the PR though. For instance, the titles should not be in italics.

Hi James, Thanks for the review. I will edit the PR.

PaulinhoNdegwa committed:
chore(heroku): deploy to heroku

- Upgrade Django 1.1 to Django 2.1
- Configure app to read database environment variables
- Create a simple default home page for API

[Finishes #163388128]

chore(heroku):Deploy to heroku

Arusey commented:
Looking good...seems everything is working fine

sirghiny commented on .travis.yml:
```diff
@@ -0,0 +1,20 @@
+language: python
```
Since travis changes are already in develop, rebase to eliminate any files already merged from the comparison.

PaulinhoNdegwa replied:
Hi @sirghiny I successfully rebased to develop.

sirghiny commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
```
I saw `.vscode` is in your `.gitignore` but the files are still here, to remove them, you can remove it from the ignored files, push a commit  to delete it then add it to the ignored files.

PaulinhoNdegwa replied:
@sirghiny I managed to get rid of the `.vscode` directory.

sirghiny commented on authors/urls.py:
```diff
@@ -13,12 +12,12 @@
+    path('admin/', admin.site.urls),
 
-    url(r'^api/', include('authors.apps.authentication.urls', namespace='authentication')),
+    # path('api/', include('authors.apps.authentication.urls', namespace='authentication')),
```
Remove all commented-out code.

PaulinhoNdegwa replied:
In a second.

PaulinhoNdegwa replied:
Done!
